### DISCUSSION TITLE
Can't set supported editor while creating new assignment

### DISCUSSION THREAD

**Discussion by hkaiser** [community/community#117212] [GitHub Education] [answered]
### Hi everyone,

While selecting a supported editor is possible while selecting the options for a new assignment, after creating it, the supported editor is not set and the page tells me that no editor was selected.

**Comment by bmlaw** [comment]
Same. Glad to hear it's not just me going crazy.

**Reply by aztharion** [reply]
Don't worry the problem will be fixed soon

**Comment by nhatcotangoc** [comment]
I'm scread

**Reply by aztharion** [reply]
Hello Don't worry

**Comment by samus-aran** [comment, marked as answer]
Hi @hkaiser 👋 

We are happy to update that we have deployed a fix for this. Can you please re-try this and confirm that it works as expected 👍

**Reply by hkaiser** [reply]
Thanks for the fix. I confirm that the problem has been resolved (as far as I can see).
